packages/internet-header/src/components/post-search/post-search.tsx

```diff
@@ -191,6 +199,17 @@ export class PostSearch implements HasDropdown, IsFocusable {
     this.deselectSuggestion();
   }
 
+  /**
+   * clear Searchbar
+   * calls handleSearchInput to update/remove suggestions
+   */
+  private handleClearSearchbar() {
```

Comment:
This is a detail but this file uses the term "searchBox" and not "searchbar". It would be nice if you could use the same term in the code you added.